tests/importerTest.php

```diff
@@ -49,52 +50,109 @@ class tool_ldapsync_importer_testcase extends advanced_testcase {
     private $sync = null;
 
     protected function setUp(): void {
+        // Create new empty test container.
+        $topdn = 'dc=moodletest,' . TEST_TOOL_LDAPSYNC_DOMAIN;
```

Comment:
I don't see `TEST_TOOL_LDAPSYNC_DOMAIN` declared anywhere. same goes for the various other `TEST_TOOL_` constants.